CONTRIBUTING.md

```diff
@@ -0,0 +1,80 @@
+# Contributing guide
+
+**Want to contribute? Great!** 
+We try to make it easy, and all contributions, even the smaller ones, are more than welcome.
+This includes bug reports, fixes, documentation, examples... 
+But first, read this page (including the small print at the end).
+
+* [Legal](#legal)
+* [Reporting an issue](#reporting-an-issue)
+* [Before you contribute](#before-you-contribute)
+  + [Code reviews](#code-reviews)
+  + [Coding Guidelines](#coding-guidelines)
+  + [Continuous Integration](#continuous-integration)
+  + [Tests and documentation are not optional](#tests-and-documentation-are-not-optional)
+  + [Current status](#current-status)
+* [The small print](#the-small-print)
+
+
+## Legal
+
+All original contributions to Apicurio projects are licensed under the
```

Comment:
I think we can change this to `Chicory` in this document

Reply:
argh, now you know where I started from 😄

Reply:
fixed